test/membrane_rtc_engine/hls_endpoint_test.exs

```diff
@@ -22,12 +22,15 @@ defmodule Membrane.RTC.HLSEndpointTest do
     [rtc_engine: pid]
   end
 
-  describe "HLS Endpoint test" do
-    @tag :skip
+  describe "HLS Endpoint" do
     test "creates correct hls stream from h264 file", %{rtc_engine: rtc_engine} do
+      # FIXME MV-147
+      # after engine refactor this test
+      # produces video with glitches
```

Comment:
According to our talk, we should check if sending video from a file with use of PayloaderBin also produces glitches.